src/scripts/main.js

```diff
@@ -354,7 +354,28 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the HTML. If it doesn't, `document.querySelector('.dashboard')` will return `null`, which will cause an error when you try to append rows to it. Consider adding a check to ensure `dashboard` is not `null` before proceeding with the loop.